### PR TITLE
Update readme and name

### LINKED ISSUE #1: Update README
To mention what this is.

### PR DESCRIPTION
**Pull request by rsheeter**
Fixes #1 and gets us building w/new name.

[rendered](https://github.com/harfbuzz/harfruzz/tree/readme)

### CODE REVIEW

**Review by cmyr — APPROVED** [review]
looks good!

File: README.md [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
 
-rustybuzz passes nearly all of harfbuzz shaping tests (2221 out of 2252 to be more precise).
-So it's mostly identical, but there are still some tiny edge-cases which
-are not implemented yet or cannot be implemented at all.
+The following conformance issues need to be fixed:
+
+* harfruzz does not yet fully pass the harfbuzz shaping or fuzzing tests
+* Malformed fonts will cause an error
+   * HarfBuzz uses fallback/dummy shaper in this case
+* No Arabic fallback shaper (requires subsetting)
```
[the remark this answers is not in the document]
Comment (rsheeter) [reply README.md:28]: Ty, will clarify in a followon

**Comment by asibahi** [comment]
Is it too late to change the name? `harfruzz` is a terrible name.

`skapa` goes with the theme of Old Norse names

**Comment by rsheeter** [comment]
I'd love a Norse name but I want to respect that this is Behdads library
and choice of name.
